topicgenerator/src/main/java/io/dockstore/topicgenerator/helper/OpenAIHelper.java

```diff
@@ -55,11 +55,11 @@ public static int countMessageTokens(EncodingRegistry registry, String model, Li
      * @param maxResponseToken
      * @return
      */
-    public static int getMaximumAmountOfTokensForUserMessageContent(EncodingRegistry registry, ModelType aiModel, ChatMessage systemMessage, int maxResponseToken) {
+    public static int getMaximumAmountOfTokensForUserMessageContent(EncodingRegistry registry, ModelType aiModel, int maxContextLength, ChatMessage systemMessage, int maxResponseToken) {
         ChatMessage userMessageWithoutContent = new ChatMessage(ChatMessageRole.USER.value());
         List<ChatMessage> messages = List.of(systemMessage, userMessageWithoutContent);
 
         final int tokenCountWithoutUserContent = countMessageTokens(registry, aiModel.getName(), messages);
-        return aiModel.getMaxContextLength() - maxResponseToken - tokenCountWithoutUserContent;
```

Comment:
Removed this because the package that provides `ModelType` has not updated the max context length for the new updates to the GPT 3.5 turbo models. It still returns a 4k context length for GPT-3.5-turbo when it should actually return 16k